src/main/java/duke/ui/DialogBox.java

```diff
@@ -0,0 +1,61 @@
+package duke.ui;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
```

Comment:
Maybe can order the import statements in alphabetical order? Although I'm not too sure of the convention, as long as your ordering of imports is consistent in all files, it should be fine!